--- kubeadm.conf.orig	2018-11-20 17:21:23.260960769 +0100
+++ kubeadm.conf	2018-11-20 17:21:45.678004579 +0100
@@ -12,7 +12,7 @@
   - authentication
 kind: InitConfiguration
 nodeRegistration:
-  criSocket: /var/run/dockershim.sock
+  criSocket: /var/run/containerd/containerd.sock
   name: ubuntu-4gb-fsn1-1
   taints:
   - effect: NoSchedule
@@ -35,7 +35,7 @@
 kubernetesVersion: v1.12.0
 networking:
   dnsDomain: cluster.local
-  podSubnet: ""
+  podSubnet: "10.32.0.0/12"
   serviceSubnet: 10.96.0.0/12
 unifiedControlPlaneImage: ""
 ---
@@ -89,7 +89,7 @@
   syncPeriod: 30s
 kind: KubeProxyConfiguration
 metricsBindAddress: 127.0.0.1:10249
-mode: ""
+mode: "ipvs"
 nodePortAddresses: null
 oomScoreAdj: -999
 portRange: ""
